packages/footer/src/sass/module.scss

```diff
@@ -0,0 +1,74 @@
+//--------------------------------------------------------------------------------------------------------------------------------------------------------------
+// footer module
+// Footers help users who reach the bottom of a page without finding what they want.
+//
+// Content:
+// - Footer styles
+//   -Basic footer block styling
+//   - Footer `nav` navigation styling (supports up to 4 link blocks)
+//   - Footer bottom section styling
+//--------------------------------------------------------------------------------------------------------------------------------------------------------------
+
+
+@import 'globals.scss';
+
+
+//--------------------------------------------------------------------------------------------------------------------------------------------------------------
+// Footer styles
+//--------------------------------------------------------------------------------------------------------------------------------------------------------------
+/**
+ * Basic footer block styling.
+ */
+.uikit-footer {
+	background-color: $uikit-colour-Background-20;
+	border-top: 6px solid $uikit-colour-Beta;
+	padding: uikit-space( 1 ) 0;
+}
+
+
+/**
+ * Footer `nav` navigation styling.
+ */
+.uikit-footer__navigation {
```

Comment:
I am very very VERY against this.
This short-hand notation will:
- help you type less
- make it easier to rename base classes and all it's siblings
- make it hard/impossible for you to search any files for the class `uikit-footer__navigation` just because we are too lazy to explicitly write it out. 
- degrades the readability of the code

On the other hand writing it out will will do two things:
- avoid side effects (explicit over implicit)
- keep your nesting/specificity down
- keep readability easy
- search is a givin
etc

Not changing. Sorry.